### ISSUE TITLE
UD WebTeam Feedback

### ISSUE THREAD

**Issue by cubap** [CenterForDigitalHumanities/DLA#37]
Accessibility

The Dunbar image in the upper right corner of all pages is missing alt text or needs to be marked decorative.
The orange links do not meet accessibility color contrast ratio: Example: index page in the Love Letters? and Every Verse, sections. The orange links appear throughout the site and on the top navigation bar that is present on the poem and letter archive pages.
The text on the "carousel like" section of each of the images on the index page is not accessible
All of the love letter images are missing alt text. 
For the love letters that only include the image of the letter, there is no text version which makes the page inaccessible for someone using a screen reader. The page would have no content that was accessible and therefore be blank.
Example: [http://dla-public.rerum.io/collection/record/618d9b9a50c86821e60b2cef](https://urldefense.com/v3/__http://dla-public.rerum.io/collection/record/618d9b9a50c86821e60b2cef__;!!K543PA!IypZ6HWpJzZuesr4nV-9-McxIIqDdVlQTUD5BSakdlZebSp9j1RsNXtUup0aCNUrjbhWKARpH4Bv4t79GjumPyGOBF67$)
The Full View Link (Gray) at the bottom of each poem does not meet contrast ratio requirements. It also goes to a blank page on each entry.

General

The filter on Love Letters and the Poems Collection doesn't work in Firefox.
The search bar doesn't work on the [index](https://urldefense.com/v3/__http://dla-public.rerum.io/index.html?__;!!K543PA!IypZ6HWpJzZuesr4nV-9-McxIIqDdVlQTUD5BSakdlZebSp9j1RsNXtUup0aCNUrjbhWKARpH4Bv4t79GjumP9EPmktR$)
Collection (redundant) and Record links in the top navigation bar are dead.


**Comment by cubap** [comment]
Search bar on index is not going to work until there is more content to index

**Comment by cubap** [comment]
Firefox Filters are troubling... The events are not bubbling correctly... this needs to be its own bug.

**Comment by cubap** [comment]
Full view is not implemented
